dokku

```diff
@@ -225,6 +227,9 @@ case "$1" in
       fi
     fi
     ;;
+  version)
```

Comment:
Could we add support for `-v` and `--version` flags as well? That would be very user-friendly :)

Reply:
:+1: done!